### PR TITLE
[ZEPPELIN-688] Giving an option to hide REPL output in Spark interpreter 

### PR DESCRIPTION
**Pull request by AhyoungRyu**
### What is this PR for?
When a user runs Spark interpreter, the result will come out with result number message like :
```
res0: Int = 250
```
Someone might want to print this REPL output with their result, but others may want to see the result only since sometimes this output is too verbose. So, I just want to give an option to hide this REPL output to those users. 

The default value of `zeppelin.spark.printREPLOutput` is `true`. This status is as-is. Users can hide REPL output only when they changes this property `true` to `false`. In this case, they can check the result by specifying such as `print(some_variable)`.

### What type of PR is it?
Improvement

### Todos
* [x] - Add a property `zeppelin.spark.printREPLOutput` 
* [x] - Add Spark interpreter property table to `docs/spark.md`

### What is the Jira issue?
[ZEPPELIN-688](https://issues.apache.org/jira/browse/ZEPPELIN-688#)

### How should this be tested?
After applying this PR,
  1. Create spark interpreter for this test and change `zeppelin.spark.printREPLOutput` property value `true` -> `false`
  2. Create a notebook and bind interpreter what you made.
  3. Write `val a = 250` down and run this paragraph. Then you can check the any output is not shown although paragraph status is **FINISHED** (This is the result of this PR).
  4. Run `print(a)` in the next paragraph. Then finally you can get a result `250`.

### Screenshots (if appropriate)
spark-result

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? I added Spark interpreter property table to `docs/spark.md`.

### CODE REVIEW

File: docs/interpreter/spark.md [hunk 1]
```diff
@@ -40,6 +40,73 @@ Spark Interpreter group, which consisted of 4 interpreters.
 </table>
 
 ## Configuration
+Zeppelin provides the below properties for Spark interpreter.
```
Comment (Leemoonsoo) [inline docs/interpreter/spark.md]: What do you think mentioning that user can add any spark properties even though they're not listed by Zeppelin?

**Comment by AhyoungRyu** [comment]
@Leemoonsoo Thanks for your suggestion. I've just added a sentence as you said : )
Plus, I rebased since #760 and this PR both changed `sparkInterpreter.java`.

File: docs/interpreter/spark.md [hunk 2]
```diff
@@ -40,6 +40,74 @@ Spark Interpreter group, which consisted of 4 interpreters.
 </table>
 
 ## Configuration
+Zeppelin provides the below properties for Spark interpreter. 
+You can also set other Spark properties which are not listed in the table. If so, please refer to [Spark Available Properties](http://spark.apache.org/docs/latest/configuration.html#available-properties).
```
Comment (AhyoungRyu) [inline docs/interpreter/spark.md]: @Leemoonsoo I added this line. Is it okay with you ?

Comment (Leemoonsoo) [inline docs/interpreter/spark.md]: Great. Thanks!

**Comment by Leemoonsoo** [comment]
Thanks. it looks good to me!

**Comment by Leemoonsoo** [comment]
Merge if there're no more discussions.

**Comment by felixcheung** [comment]
LGTM.

**Pull request closed without merging**